.github/workflows/apm-capabilities.yml

```diff
@@ -6,13 +6,22 @@ on:
     branches: [master]
   schedule:
     - cron: 0 4 * * *
+  workflow_dispatch:
+    inputs:
+      nightly-version:
```

Comment:
Similar to the above, this should be something like `latest-version`. The fact that a nightly can be passed in is an implementation detail.